Framework/Kernel/test/PropertyManagerTest.h

```diff
@@ -481,6 +481,15 @@ class PropertyManagerTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(i, 655);
   }
 
+  void test_char_array() {
```

Comment:
Actually I think it's a good test if the proper asserts are added. I hadn't seen a test that tested it throws in this situation?